File: RoRClient/Commands/Editor/Other/ChangeMapNameCommand.cs
using RoRClient.Commands.Base;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using RoRClient.Communication.DataTransferObject;
using RoRClient.Models.Session;

namespace RoRClient.Commands.Editor.Other
{
	class ChangeMapNameCommand : CommandBase
	{
		private string mapName;
		public ChangeMapNameCommand(RoRSession session, MessageInformation message) : base(session, message)
		{
			mapName = message.GetValueAsString("mapName");
		}
		public override void Execute()
		{
			session.MapName = mapName;
		}

	}
}

File: RoRServer/src/main/java/commands/editor/ChangeMapNameCommand.java
package commands.editor;

import commands.base.CommandBase;
import communication.MessageInformation;
import models.session.EditorSession;
import models.session.RoRSession;

/**
 * Setzt den neuen MapName der GameSession
 */
public class ChangeMapNameCommand extends CommandBase {

	private String mapName;

	public ChangeMapNameCommand(RoRSession session, MessageInformation messageInfo) {
		super(session, messageInfo);
		this.mapName = messageInfo.getValueAsString("mapName");
	}

	@Override
	public void execute() {
		EditorSession editorSession = (EditorSession) session;
		editorSession.setMapName(mapName);
	}
}
